larsim/EventGenerator/POTaccumulator_module.cc

```diff
@@ -0,0 +1,307 @@
+/**
+ * @file   larsim/EventGenerator/POTaccumulator_module.cc
+ * @brief  Module summing all POT from the input files.
+ * @date   April 29, 2020
+ * @author Gianluca Petrillo (petrillo@slac.stanford.edu)
+ */
+
+// LArSoft libraries
+#include "larcorealg/CoreUtils/get_elements.h"
+#include "larcoreobj/SummaryData/POTSummary.h"
+
+// framework libraries
+#include "art/Framework/Core/EDAnalyzer.h"
+#include "art/Framework/Core/ModuleMacros.h"
+#include "art/Framework/Principal/SubRun.h"
+#include "art/Framework/Principal/Handle.h"
+#include "canvas/Utilities/InputTag.h"
+#include "fhiclcpp/types/Atom.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+
+// C/C++ standard libraries
+#include <map>
+#include <string>
+#include <algorithm> // std::for_each()
+
+
+// -----------------------------------------------------------------------------
+namespace sim { class POTaccumulator; }
+
+/**
+ * @brief  Prints on console the total Protons On Target from the input subruns.
+ *
+ * This module collects information from each of the subrun in the input files
+ * and prints the total of the protons on target. It also optionally prints the
+ * total for each observed run.
+ * If a subrun is met more than once, the information from all subrun fragments
+ * are added together (i.e. it is assumed that summary information is
+ * complementary rather than duplicate).
+ *
+ * The output is printed to the console via message facility.
```

Comment:
As you know, the message facility can be configured so that nothing is printed to the console.  Should adjust to say "The output can be printed to the console via the appropriate message facility configuration."

Reply:
Implemented.